backend/src/models/dto/new_seed_dto.rs

```diff
@@ -0,0 +1,56 @@
+use crate::models::{r#enum::quality::Quality, seed::Seed};
+use crate::models::r#enum::quantity::Quantity;
+use crate::models::r#enum::tag::Tag;
+use bigdecimal::BigDecimal;
+use chrono::NaiveDate;
+use serde::{Deserialize, Serialize};
+use typeshare::typeshare;
+
+#[typeshare]
+#[derive(Serialize, Deserialize)]
+pub struct NewSeedDTO {
+    pub id: Option<i32>,
+    pub name: String,
+    pub variety_id: i32,
```

Comment:
Why signed?

Reply:
oh yeah it should be unsigned, I'm not used to having control over signed/unsigned integers

Reply:
Diesel expects signed integers and gives me errors if I try to change it to unsigned

https://docs.rs/diesel/2.0.2/diesel/sql_types/struct.SmallInt.html